package.json

```diff
@@ -17,6 +17,7 @@
     "start": "next start"
   },
   "dependencies": {
+    "@react-spring/three": "^9.2.4",
```

Comment:
Ah, I did not think that test_shader would be added to the typescript.
As it's a starter we should let the user decide to include @react-spring/three or not. So the best would be to remove the @react-spring dependency.

Instead of :
`scale={scale.to((s) => [s, s, s])}`
we should do this:
`scale={hovered ? 7 : 5}`

Reply:
> As it's a starter we should let the user decide to include @react-spring/three or not. So the best would be to remove the @react-spring dependency.

Okay, I'll fix it!